editions/ja-JP/tiddlywiki.info

```diff
@@ -30,8 +30,8 @@
 			"--rendertiddler","$:/editions/ja-JP/download-empty","empty.html","text/plain",
 			"--rendertiddler","$:/editions/ja-JP/download-empty","empty.hta","text/plain"],
 		"favicon": [
-			"--savetiddler","$:/favicon.ico","favicon.ico",
```

Comment:
The "./favicon.ico" was historically hardwired into browsers, and so for backwards compatibility I'd be inclined to keep the `favicon.ico` output filename for the PNG file.